config/bootstrap.php

```diff
@@ -162,10 +162,12 @@
  */
 Request::addDetector('mobile', function ($request) {
     $detector = new \Detection\MobileDetect();
+
     return $detector->isMobile();
 });
 Request::addDetector('tablet', function ($request) {
```

Comment:
Instead of inside, there should more importantly be a newline after each closure..
Now it looks like the return belongs to the next statement etc more than it belongs to the actual own closure.